Sources/AWSLambdaRuntimeCore/LambdaContext.swift

```diff
@@ -68,11 +70,17 @@ extension Lambda {
         /// For invocations from the AWS Mobile SDK, data about the client application and device.
         public let clientContext: String?
 
+        /// Context baggage.
+        public var baggage: BaggageContext
+
         /// `Logger` to log with
         ///
         /// - note: The `LogLevel` can be configured using the `LOG_LEVEL` environment variable.
         public let logger: Logger
 
+        /// Tracing instrument.
+        public let tracer: TracingInstrument
```

Comment:
Not sure about the general pattern of storing a tracer, but also not really against it, let's see how it goes -- in lambda specifically I agree it can be nice to have the tracer just available like this, makes adding spans by users much easier 👍